src/type-coercions.md

```diff
@@ -172,6 +172,8 @@ an implementation of `Unsize<U>` for `T` will be provided:
 
 * `T` to `dyn U`, when `T` implements `U + Sized`, and `U` is [object safe].
 
+* `dyn T` to `dyn U`, when `T` has `U` as one of its ancestor trait.
```

Comment:
```suggestion
* `dyn T` to `dyn U`, when `U` is one of `T`'s supertraits.
```

Reply:
I don't think we use "ancestor traits" conventionally anywhere in the compiler, so I think we should prefer a term that *is* used in the compiler.